src/Api/Order.php

```diff
@@ -92,6 +92,13 @@ public function endpoint_args() {
 					return false === $token_id || is_string( $token_id );
 				},
 			],
+			'user_email' => [
```

Comment:
That's correct, but the information of the order, it works in the same way as the token.

We can use the user ID but I think that's more risky to use. Let me know what do you think, thanks.